docs/articles/roadmap/README.md

```diff
@@ -15,18 +15,16 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Visual Regression Testing](https://github.com/DevExpress/testcafe/issues/1207)                   | Under Consideration
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Under Consideration
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | Planned
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
```

Comment:
Is it really `HTTP API Testing` - not `REST API Testing` or something?

Reply:
REST API testing is a subset of API Testing. I think API testing is the more common term that HTTP API testing..

Reply:
API testing is very common term included load, security and etc. (https://en.wikipedia.org/wiki/API_testing).
In my opinion, HTTP API testing or Web API testing is best variant.

Reply:
Of course it is, but in other products it named just API testing.

Reply:
Ok.